wallet/src/wallet/schema.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE IF NOT EXISTS "utxos" (
+  "id"             integer     PRIMARY KEY,
+  "txid"           text        NOT NULL,
+  "vout"           integer     NOT NULL,
+  "address"        text        NOT NULL REFERENCES "address" ("id"),
+  "value"          integer     NOT NULL,
+  "date"           integer     NOT NULL,
+
+  UNIQUE ("txid", "vout")
+) STRICT;
+
+CREATE TABLE IF NOT EXISTS "addresses" (
+  "id"          text             PRIMARY KEY,
+  "index"       integer          NOT NULL UNIQUE,
+  "label"       text             DEFAULT NULL,
+  "received"    integer          NOT NULL DEFAULT 0,
+  "used"        integer          NOT NULL DEFAULT false
+) STRICT;
```

Comment:
```suggestion
CREATE TABLE IF NOT EXISTS "utxos" (
  "id"             integer     PRIMARY KEY,
  "txid"           text        NOT NULL,
  "vout"           integer     NOT NULL,
  "address"        text        NOT NULL REFERENCES "address" ("id"),
  "value"          integer     NOT NULL,
  "date"           integer     NOT NULL,

  UNIQUE ("txid", "vout")
);

CREATE TABLE IF NOT EXISTS "addresses" (
  "id"          text             PRIMARY KEY,
  "index"       integer          NOT NULL UNIQUE,
  "label"       text             DEFAULT NULL,
  "received"    integer          NOT NULL DEFAULT 0,
  "used"        integer          NOT NULL DEFAULT false
);
```

It is possible remove the STRIC syntax from there? it is supported only in sqlite > 3.37 and this can bring problem in system that has the version below these

```
I did not going in deep, but while populating the db I received the following error

```
➜  nakamoto git:(a5530a8) ✗ sqlite3 wallet.db < insert-addresses.sql                                                                                    
Error: near line 1: malformed database schema (utxos) - near "STRICT": syntax erro
```

In the other hand, in this case, we a lost type safety from sqllite right?

Reply:
So normally sqlite3 should be statically linked, because of this line in Cargo.toml:
```
sqlite3-src = { version = "0.4.0", features = ["bundled"] }
```
On my system this works, ie. there's no external dependency to my system sqlite3.

Reply:
Yeah, this makes sense, the insert in the db it is not usual! I think here the problem is that the nakamoto use the right version but my system no, and I can not make the insert!

Reply:
Ah, strange, so the STRICT affects the insert as well?

Reply:
yes, when I ran the command `sqlite3 wallet.db < insert-addresses.sql`

Reply:
Gotcha, then you'll have to update your sqlite3 or write a rust script to insert the addresses.

Reply:
Yeah, I think the rust script is the way to go, because updating sqlite3 is a little bit tricky, I have some apps that relay on it.

But it is not a big deal :)